app/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "react boilerplate",
+  "short_name": "react bp", // 12 chars or less
```

Comment:
@gretzky thank you for this :)

one nit: would you mind removing inline comment (since not JSON spec compliant)

maybe we could explain this constraint in `general/gotchas.md`?
https://github.com/react-boilerplate/react-boilerplate/blob/dev/docs/general/gotchas.md

ie:

```
## Using short_name in Manifest.json (PWA)

When there's insufficient space to display an app's full name it is truncated.
This happens with app launcher and new tab in Chrome for Android.
The `short_name` field allows a *12 character or less abbreviation*.
It also addresses any problems in testing the PWA in Lighthouse.
```

Reply:
Done and done!